packages/visual-editor/src/store/tree.ts

```diff
@@ -111,3 +156,7 @@ export const useTreeStore = create<TreeStore>((set, get) => ({
     );
   },
 }));
+
+function cloneDeepAsPOJO(obj) {
```

Comment:
As POJO might be a bit unclear to other devs, I would like to explain what the intention behind this helper is:
```suggestion
// Serialize and deserialize an object again to remove all functions and references.
// Some people refer to this as "Plain Old JavaScript Object" (POJO) as it solely contains plain data.
function cloneDeepAsPOJO(obj) {
```